readme.md

```diff
@@ -0,0 +1,237 @@
+# meta-xt-prod-devel
+
+## Content
+This document is long enough to has it's own content. So below you will find:
+* What is meta-xt-prod-devel
+* Quick start
+  * Some basic and most used commands
+* Usage and setup
+  * Serial over USB
+  * SSH over ethernet
+  * What to do "on board"
+* How to build
+  * Few build hints
+* How to install
+  * Write to microSD card
+  * Write to eMMC
+* How to prepare board (loaders and u-boot settings)
+* Supported boards
+* References
+
+Important note.
+All PC-related steps described bellow are intended to be run on Linux, and were tested on Ubuntu 18.
+Windows and MacOS are not supported, sorry.
+
+## What is meta-xt-prod-devel
+
+This is meta layer that allows you to run Linux+Linux or Linux+Android at same time under Xen hypervisor on some Renesas' R-CAR boards.
```

Comment:
please add **Gen3** after Renesas' to be more specific.

Reply:
Gen3 added.